### PR TITLE
[Changelog]: Updating to 1.4.7

### PR DESCRIPTION
**Pull request by antross**
VSIX to be published (zipped to allow upload to GitHub): [vscode-edge-devtools-1.4.7.zip](https://github.com/microsoft/vscode-edge-devtools/files/8422064/vscode-edge-devtools-1.4.7.zip)

### CODE REVIEW

**Comment by antross** [comment]
@mliao95 @hkal, I have this PR targeting a new servicing branch I created where we can cherry-pick any fixes we want to release before the next major update is ready.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.4.7
+* **[Bug Fix]**: Update vscode-webhint to version 1.6.6. Fixes "Unable to start webhint" errors - [PR #951](https://github.com/microsoft/vscode-edge-devtools/pull/951)
+* **[Bug Fix]**: Improve logic for configuring existing launch.json - [PR #945](https://github.com/microsoft/vscode-edge-devtools/pull/
```
Comment (mliao95) [inline CHANGELOG.md:3]: Missing closing parentheses and pull request number

**Review by mliao95 — APPROVED** [review]
LGTM after minor change

File: CHANGELOG.md (hunk 1 above)
Comment (antross) [reply CHANGELOG.md]: Good catch! Text is fixed and linked VSIX updated (since the VS Code UI uses the changelog from the package).

**Pull request merged**